### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by jjl014**
Looks like there have been some changes to the API recently causing some of the tests to fail. All tests should pass now with these changes.

### Updates:
* Add default section value to the items api v2 functions
* Minor change to test assertions for `fmt_options`

Screen Shot 2023-01-18 at 4 15 09 PM
Screen Shot 2023-01-18 at 4 21 04 PM

### CODE REVIEW

File: spec/src/modules/catalog/catalog-files.js [hunk 1]
```diff
@@ -210,7 +210,6 @@ describe('ConstructorIO - Catalog', () => {
           variations: variationsBuffer,
           item_groups: itemGroupsBuffer,
           section: 'Products',
-          force: true,
```
Comment (jjl014) [inline spec/src/modules/catalog/catalog-files.js:213]: I added this previously in a separate PR and forgot to remove it.

Comment (esezen) [reply spec/src/modules/catalog/catalog-files.js:213]: I saw this on the other PR but thought you left it there intentionally to reset the catalog.

**Review by esezen — APPROVED** [review]
LGTM! We didn't define a default for `section` initially to leave that responsibility to the API (like if they change it, we don't have to) but it looks like they require this now. Started an internal thread and tagged you. If they decide that wasn't intentional and revert back to not requiring it, I think we can close this without merging.

**Comment by jjl014** [comment]
@esezen I removed the section changes and made some small adjustments to fix other failing tests. All the tests should be passing just fine now 👍

**Review by esezen — APPROVED** [review]
LGTM!

**Pull request merged**